src/webparts/laserficheAdminConfiguration/components/ManageMappingsPage/ManageMappingsPage.tsx

```diff
@@ -584,7 +584,7 @@ export default function ManageMappingsPage(
               {lfContentTypesDisplay}
             </select>
           </td>
-          <td className={styles.iconTableCell}>
+          <td>
```

Comment:
what is the effect of removing the iconTableCell class and adding the dataCellWidth class?

Reply:
iconTableCell was doing vertical-align: middle. After finding myself doing this somewhere else, I made the style on all td cells in the component, not just ones with the class
dataCellWidth made those cells 50% of the width since there are two columns